sdk/js/src/Client.ts

```diff
@@ -123,6 +123,11 @@ export class DVCClient implements Client {
                     ? EventTypes.variableDefaulted
                     : EventTypes.variableEvaluated,
                 target: variable.key,
+                metaData: {
+                    value: variable.value,
+                    type: typeof(variable.defaultValue),
+                    _variable: this.config?.variables?.[key]._id
```

Comment:
can probably use the existing data object defined on line 106?
```suggestion
                    _variable: data._id
```